concent_api/core/tests/utils.py

```diff
@@ -13,6 +13,7 @@
 
 from golem_messages         import dump
 from golem_messages         import load
+from golem_messages         import cryptography
 from golem_messages         import message
```

Comment:
We should remember about the alphabetical order.

Reply:
Done